README.md

```diff
@@ -2,6 +2,7 @@
 
 [![License badge](https://img.shields.io/badge/license-GPL-blue.svg)](https://opensource.org/licenses/GPL-3.0)
 [![Docker badge](https://img.shields.io/docker/pulls/dojot/iotagent-json.svg)](https://hub.docker.com/r/dojot/device-manager/)
+[![Travis badge](https://travis-ci.org/dojot/device-manager.svg?branch=master)](https://travis-ci.org/dojot/device-manager#)
```

Comment:
mmm, we're actually just building docs, so this can be misleading - but ok

Reply:
Travis has built the docs correctly :)
I agree - this badge should be here only if there's something that could go wrong and affect others.